docs/docs/README.md

```diff
@@ -0,0 +1,581 @@
+    Adding uberjar entries...
+    Writing pom.xml and pom.properties...
+    Writing zero-20190409-5.jar...
+    Writing target dir(s)...
+    tbl@wm97a-c2b ~/Broad/zero # java -jar ./target/zero-20190409-5.jar
+    ...
+    tbl@wm97a-c2b ~/Broad/zero 1#
+    ```
+
+5.  Rich Comments
+    
+    Some Clojure source files have `(comment ...)` forms at the
+    bottom.
+    
+    ``` example
+    tbl@wm97a-c2b ~/Broad/zero # tail ./src/zero/db.clj ./src/zero/main.clj
+    ==> ./src/zero/db.clj <==
+                  {:connection-uri (metrics-sql-url environment)}
+                  db-spec)
+                :user username :password password) sql)))
+    
+    (comment
+      (query [:on-prem-picard :dev]
+              "select count (*) from picard.res_proj_agg_override")
+      (query [:cloud-metrics :dev]
+              "SELECT COUNT(*) FROM EXOME_METRICS")
+      )
+    
+    ==> ./src/zero/main.clj <==
+      (-main "write-inputs")
+      (-main "write-inputs" "WF=ExomeGermlineSingleSample" "FGBN=FGBN" "S=S"
+              "REF=./reference.json" "CON_REF=./reference_contamination.json"
+              (str "UBAMS=" ubam))
+      (-main "write-inputs" "WF=ExomeGermlineSingleSample" "FGBN=FGBN" "S=S"
+              "REF=./reference.json" "CON_REF=./reference_contamination.json"
+              (str "UBAMS=" ubam) "O=./o.json")
+      (-main "run-starter" "ENV=dev" "VERBOSITY=fnord" "fnord")
+      (-main "run-starter" "EMAIL=tbl@broadinstitute.org")
+      )
+    tbl@wm97a-c2b ~/Broad/zero #
+    ```
+    
+    They permit fast testing of code changes by storing expressions
+    that you can evaluate in your editor buffer.
+    
+    Feel free to add, edit, or augment them as you see fit.
+
+6.  Documentation
+    
+    This file is written in markdown.
+
+8.  Exomes in the Cloud Resources
+    
+    From [Hybrid Selection in the Cloud
+    V1](https://docs.google.com/a/broadinstitute.org/document/d/1g8EmPjOZl-DzHlypXeOjKHzI4ff1LvzBiigDbZTy1Cs/edit?usp=sharing)
+    
+    1.  Clients
+        
+          - [Google Cloud Storage Client Library
+            (Java)](https://cloud.google.com/storage/docs/reference/libraries#client-libraries-install-java)
+        
+          - [Google Cloud Client Library for
+            Java](https://googlecloudplatform.github.io/google-cloud-java/0.30.0/index.html)
+    
+    2.  Diagrams
+        
+          - [Zamboni
+            Overview](https://confluence.broadinstitute.org/download/attachments/39552724/ZamboniOverview.pdf)
+    
+    3.  Sources
+        
+          - /Users/tbl/Broad/zamboni/Client/src/scala/org/broadinstitute/zamboni/client/lightning/clp/Lightning.scala
+          - /Users/tbl/Broad/picard-private/src/java/edu/mit/broad/picard/lightning
+          - /Users/tbl/Broad/gppipeline-devtools/release<sub>client</sub>
+          - /Users/tbl/Broad/gppipeline-devtools/starter<sub>control</sub>
+          - /picard02:/seq/pipeline/gppipeline-devtools/current/defs/prod.defs
+
+#### Cruft
```

Comment:
Should this section be removed?

Reply:
Good catch, we don't support sv anymore, so makes sense to remove this.